etc/ComCam/primary_hdu.header

```diff
@@ -108,4 +108,7 @@ HIERARCH SIMULATE CAMHEXAPOD= / CAMHexapod Simulation Mode (False=0)
 HIERARCH SIMULATE M2HEXAPOD= / M2Hexapod Simulation Mode (False=0)
 HIERARCH SIMULATE MTROTATOR= / MTRotator Simulation Mode (False=0)
 HIERARCH SIMULATE MTDOME= / MTDome Simulation Mode (False=0)
-HIERARCH SIMULATE MTDOMETRAJECTORY= / MTDomeTrajectory Simulation Mode (False=0)
```

Comment:
Did you mean for the comment to get removed?

Reply:
Nope, thanks for noticing!